test/shared/Planner.sol

```diff
@@ -30,16 +32,29 @@ library Planner {
         actions[actions.length - 1] = action;
         params[params.length - 1] = param;
 
-        return Plan({actions: actions, params: params});
+        plan.actions = actions;
+        plan.params = params;
+
+        return plan;
     }
 
     function finalize(Plan memory plan, PoolKey memory poolKey) internal pure returns (bytes memory) {
```

Comment:
imo this should now be called `finalizeModifyLiquidity` or something? To differentiate between what `finalizeSwap` and `finalize` are